pkg/retry/azure_error.go

```diff
@@ -262,22 +262,19 @@ func getRetryAfter(resp *http.Response) time.Duration {
 	return dur
 }
 
-// GetErrorWithRetriableHTTPStatusCodes gets an error with RetriableHTTPStatusCodes.
+// ContainsRetriableHTTPStatusCodes return true when status code is retriable
 // It is used to retry on some HTTPStatusCodes.
-func GetErrorWithRetriableHTTPStatusCodes(resp *http.Response, err error, retriableHTTPStatusCodes []int) *Error {
-	rerr := GetError(resp, err)
+func ContainsRetriableHTTPStatusCodes(rerr *Error, retriableHTTPStatusCodes []int) bool {
```

Comment:
Can it be renamed to `ContainsHTTPStatusCode` (more generic and simpler)? Since it depends on the given `httpStatusCodes`, it doesn't care if it's not retriable.

Reply:
updated